docs/first-order/receptive_fields.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Showing Receptive Fields\n",
```

Comment:
<p>It might be helpful to add why you use Gabor patches for receptive field mapping here. (e.g. something like "Gabor patches are used to reliably drive neural responses.")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>